### PR TITLE
Feature/dtm 10.15.0.0 release notes

### PR DESCRIPTION
**Pull request by spau6**

### CODE REVIEW

**Comment by BeateRixen** [comment]
@spau6 Please move the entire text on DTM into the Whats New section of the 10.15 release notes. There is no need for separate release notes as this is the first version of DTM In Cumulocity IoT.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 1]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
 In an effort to provide you a more consistent experience across the platform the **All devices** page has been re-shaped. It is now implemented in the data grid fashion you already are familiar with from **Subassets** and **Search** pages. This offers improved appearence in various viewport sizes, easier filtering and sorting, pagination option (only for users with global inventory access) and sets solid ground for further improvements.
 
 ![All devices](/images/release-notes/devmgmt-devices-alldevices.png)
+
+### Digital Twin Manager
+
+Cumulocity IoT Digital Twin Manager is a layered product hosted on Cumulocity platform. The Digital Twin Manager enables you to create and manage assets around your physical connected devices in Cumulocity IoT. Assets can be used to structure and describe devices in logical hierarchies as encountered in the real world environment.
+
+The Digital Twin Manager consists of several components, which support you to create your assets and asset hierarchies:
```
Comment (MWindel) [inline content/release-10-15-0/whatsnew-10-15-0.md:19]:
```suggestion
The DTM consists of several components, which allow you to create your assets and asset hierarchies:
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 2]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+
+    •	Property library - Create your own custom properties (simple or complex) to define how data should be stored on an asset
```
Comment (MWindel) [inline content/release-10-15-0/whatsnew-10-15-0.md:21]:
```suggestion
    •	**Property library** - Create your own custom properties (simple or complex) to define how data should be stored on an asset
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 3]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+    •	Asset types - Define asset types to describe the blueprint of your assets
+    •	Localization - Dynamically translate asset type and custom property descriptions to multiple languages
+    •	Assets navigator - Manage your asset hierarchies
+
+You can access Digital Twin Manager from the Cumulocity IoT app switcher:
+
+![App switcher](/images/dtm/app-switcher-dtm.png)
```
Comment (MWindel) [inline content/release-10-15-0/whatsnew-10-15-0.md:28]:
```suggestion
![Application Switcher](/images/dtm/app-switcher-dtm.png)
```

Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:28]: This image is not of much value. I would better show the DTM here.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 4]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+
+**Assets Navigator**
+
+The Assets Navigator is the starting point to create your asset hierarchies based on the asset types and custom properties, you have defined previously. Once you have created your asset hierarchies you can use the Assets Navigator to view and manage them.
+
+**Localization**
+
+You can use the Localization component to define translations for your asset types and custom properties. These translations can be provided for all supported languages in Cumulocity IoT. This component requires to have the feature-branding feature subscribed for your Cumulocity tenant.
+
+### Installation
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:48]: Needs one more # to have the correct heading level

Comment (spau6) [reply content/release-10-15-0/whatsnew-10-15-0.md]: Added one more #

File: content/release-10-15-0/whatsnew-10-15-0.md (hunk 3 above)
Comment (spau6) [reply content/release-10-15-0/whatsnew-10-15-0.md]: Is it DTM landing screen?

**Comment by MWindel** [comment]
@spau6 The screenshot needs to be edited according to our guidelines.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 5]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+Cumulocity IoT Digital Twin Manager (DTM) is a layered product hosted on the Cumulocity IoT platform. The DTM enables you to create and manage assets mirroring your physical connected devices in Cumulocity IoT. These assets are then used to structure and describe devices in logical hierarchies as encountered in the real world environment.
+
+The DTM consists of several components, which allow you to create your assets and asset hierarchies:
+
+    •	**Property library** - Create your own custom properties (simple or complex) to define how data should be stored on an asset
+    •	**Asset types** - Define the asset types to describe the blueprint of your assets
+    •	**Localization** - Dynamically translate the asset type and custom property descriptions to multiple languages
+    •	**Assets navigator** - Manage your asset hierarchies
+
+You can access the DTM from the Cumulocity IoT Application Switcher:
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:26]:
```suggestion
You can access the DTM from the Cumulocity IoT application switcher:
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 6]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+
+![Application Switcher](/images/dtm/app-switcher-dtm.png)
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:28]:
```suggestion
![Application switcher](/images/dtm/app-switcher-dtm.png)
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 7]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+
+See the [Cumulocity IoT DTM Guide](/guides/dtm/#overview) for the complete documentation.
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:30]:
```suggestion
See the [Cumulocity IoT DTM guide](/guides/dtm/#overview) for the complete documentation.
```

File: content/release-10-15-0/whatsnew-10-15-0.md (hunk 3 above)
Comment (BeateRixen) [reply content/release-10-15-0/whatsnew-10-15-0.md]: Yes, I think that is better. Showing the app switcher does not make much sense.

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 8]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+    •	**Property library** - Create your own custom properties (simple or complex) to define how data should be stored on an asset
+    •	**Asset types** - Define the asset types to describe the blueprint of your assets
+    •	**Localization** - Dynamically translate the asset type and custom property descriptions to multiple languages
+    •	**Assets navigator** - Manage your asset hierarchies
+
+You can access the DTM from the Cumulocity IoT application switcher:
+
+![Application switcher](/images/dtm/app-switcher-dtm.png)
+
+See the [Cumulocity IoT DTM guide](/guides/dtm/#overview) for the complete documentation.
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:30]:
```suggestion
For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overview).
```

File: content/release-10-15-0/whatsnew-10-15-0.md [hunk 9]
```diff
@@ -11,3 +11,40 @@ Release 10.15.0 includes the following new features or major feature enhancement
...
+
+The **Assets Navigator** is the starting point to create your asset hierarchies based on the asset types and custom properties, you have defined previously. Once you have created your asset hierarchies you can use the **Assets Navigator** to view and manage them.
+
+**Localization**
+
+You can use the **Localization** component to define translations for your asset types and custom properties. These translations can be provided for all languages supported in Cumulocity IoT. This component requires you to have the feature-branding feature subscribed for your Cumulocity IoT tenant.
+
+#### Installation
+
+See the [Cumulocity IoT DTM Installation & operations guide](<<TBD>>) for the complete documentation.
```
Comment (BeateRixen) [inline content/release-10-15-0/whatsnew-10-15-0.md:50]:
```suggestion
For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.
```

Comment (BeateRixen) [reply content/release-10-15-0/whatsnew-10-15-0.md:50]: We can not link to the installation documentation as these are only available as PDF.

Comment (spau6) [reply content/release-10-15-0/whatsnew-10-15-0.md]: Do we need to mention all the installation steps which we have documented in Ops guide?
Re writing the same content is fine?

File: content/release-10-15-0/whatsnew-10-15-0.md (hunk 3 above)
Comment (spau6) [reply content/release-10-15-0/whatsnew-10-15-0.md]: Then I will add the DTM landing page screenshot

Comment (spau6) [reply content/release-10-15-0/whatsnew-10-15-0.md]: I have updated the image and committed. Please review and let me know if anything needs to be modified

**Comment by MWindel** [comment]
@spau6 please crop the screenshot to remove unnecessary white space

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@MWindel How about the article with DTM? Please change if there should not be an article. My take is to not use an article here. We also do not use an article with MLW (Machine Learning Workbench), we only us it if we say "the MLW application", for example.

**Pull request merged**